apps/etterlatte-vilkaarsvurdering/src/main/kotlin/no/nav/etterlatte/vilkaarsvurdering/AldersovergangService.kt

```diff
@@ -23,6 +23,9 @@ class AldersovergangService(
             vilkaarsvurderingService.hentVilkaarsvurdering(behandlingId)
                 ?: vilkaarsvurderingService.kopierVilkaarsvurdering(behandlingId, loependeBehandlingId, brukerTokenInfo)
 
+        // Resette
```

Comment:
trenger vi kommentaren her, er funksjonsnavnet for dårlig kanskje?

Reply:
Gjorde dette litt kjapt pga feilen i dev, men tror jeg heller utvider _kopierVilkaarsvurdering_ med et parameter for resultatet i stedet, så slipper man ekstra behandlingsstatus-endring osv.

Reply:
oppdatert iht ☝️